tests/node-sync/run_bridge.go

```diff
@@ -139,13 +137,17 @@ func RunBridgeNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 		return err
 	}
 
-	runenv.RecordMessage("Finished published bridgeID Addr %d", int(initCtx.GlobalSeq))
+	runenv.RecordMessage("Finished published bridgeID Addr %d", int(initCtx.GroupSeq))
 
-	eh, err = nd.HeaderServ.GetByHeight(ctx, uint64(8))
+	eh, err = nd.HeaderServ.GetByHeight(ctx, uint64(13))
```

Comment:
why 13?

Reply:
Waiting for the light/full to get to 12.
I'll change the number to a runenv value